apps/meteor/server/models/raw/LivechatRooms.js

```diff
@@ -1288,4 +1288,39 @@ export class LivechatRoomsRaw extends BaseRaw {
 			},
 		]);
 	}
+
+	// These 3 methods shouldn't be here :( but current EE model has a meteor dependency
+	// And refactoring it could take time
+	setTranscriptRequestedPdfById(rid) {
```

Comment:
Can you re-use this one? At least based on the imports it doesn't seem to have meteor dependency 
https://github.com/RocketChat/Rocket.Chat/blob/7440e030f1551515b4ec29ff5fd8a6eed894d4a1/apps/meteor/ee/server/models/raw/LivechatRooms.ts#L26

Reply:
"but current EE model has a meteor dependency"

It has one, on the query helper, on the units call 😬 and we haven't refactored it yet. So while we don't do it, i can't use that model (and refactoring is OOS for now)